libmachine/drivers/rpc/client_driver.go

```diff
@@ -173,12 +174,6 @@ func (c *RPCClientDriver) Close() error {
 	c.heartbeatDoneCh <- true
 	close(c.heartbeatDoneCh)
 
-	log.Debug("Making call to close connection to plugin binary")
-
-	if err := c.plugin.Close(); err != nil {
-		return err
-	}
-
 	log.Debug("Making call to close driver server")
 
 	if err := c.Client.Call(CloseMethod, struct{}{}, nil); err != nil {
```

Comment:
This RPC call should happen _first_, because otherwise we will actually exit the process and not be able to successfully make the call to the server.